<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>418</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微軟正黑體</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>401</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_multi_download">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <attribute name="title">
      <string>YT Link</string>
     </attribute>
     <widget class="QLabel" name="lb_link1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Link-1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_link1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>371</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Type Link Here</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_link2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>371</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Type Link Here</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_link2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Link-2</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_link3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Link-3</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_link3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>371</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Type Link Here</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_link4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Link-4</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_link4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>371</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Type Link Here</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_link6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Link-5</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_link5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>371</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Type Link Here</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_playlist_download">
     <attribute name="title">
      <string>YT Playlist</string>
     </attribute>
     <widget class="QLineEdit" name="le_link11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>371</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Type Link Here</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_link11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Link-Playlist</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="le_download_path">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_select_path">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PATH</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_download_path">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Download Path :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_download">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>300</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Click Me to Download YouTube</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>161</width>
      <height>79</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lb_filetype">
       <property name="text">
        <string>File Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_mp3">
       <property name="text">
        <string>MP3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_mp4">
       <property name="text">
        <string>MP4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>160</width>
      <height>108</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lb_quality">
       <property name="text">
        <string>Quality</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_high">
       <property name="text">
        <string>Highest</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_1080">
       <property name="text">
        <string>1080P</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_720">
       <property name="text">
        <string>720P</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
